test/API/H5_api_dataset_test.c

```diff
@@ -4002,13 +3936,6 @@ test_dataset_property_lists(void)
         goto error;
     }
 
-    /** for DAOS VOL, this test is problematic since auto chunking can be selected, so skip for now */
-    if (H5VLget_connector_name(file_id, vol_name, 5) < 0) {
```

Comment:
With these changes we should be able to skip tests that are problematic for particular VOL connectors